paddlebrick.p8

```diff
@@ -12,6 +12,37 @@ flags = {
   power_up = 5,
 }
 
+animations = {
+  destroy_block = {
+    active = false,
+    frames = {49, 50, 51, 52},
+    play_location_x = 0,
+    play_location_y = 0,
+    duration = 40
+  }
+}
+
+function animations.destroy_block:play()
+  -- todo: this looks ok, but we may
```

Comment:
Yeah, this TODO is pretty big. We need to create a method that returns new instances. Like a constructor, only in this crappy language the syntax sucks.